### PR TITLE
Fix wording in introduction.md

### PR DESCRIPTION
**Pull request by patrickjaberg**
The change is probably best described by the diff.  :-)

### CODE REVIEW

**Review by lpil — COMMENTED** [review]
This was right before I'm afraid. There are 2 functions that can be used to create Regex objects

**Comment by evanasse** [comment]
Should it be "The regular expression creation functions return an error...", then?
The word "return" is missing in the sentence.

File: exercises/concept/log-parser/.docs/introduction.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The most common way to create regular expressions is using the `regex.from_strin
 let assert Ok(re) = regex.from_string("test")
 ```
 
-The regular expression creation functions an error if the regular expression syntax is invalid, so a let-assertion has been used here to ensure the regular expression is valid.
+The regular expression creation function returns an error if the regular expression syntax is invalid, so a let-assertion has been used here to ensure the regular expression is valid.
```
Comment (lpil) [inline exercises/concept/log-parser/.docs/introduction.md:21]:
```suggestion
The regular expression creation functions return an error if the regular expression syntax is invalid, so a let-assertion has been used here to ensure the regular expression is valid.
```

**Review by lpil — APPROVED** [review]
Thank you both!

**Pull request merged**